sdk/communication/Azure.Communication.CallingServer/src/Models/DtmfOptions.cs

```diff
@@ -21,6 +21,6 @@ public DtmfConfigurations()
         /// <summary> Maximum number of DTMFs to be collected. </summary>
         public int? MaxTonesToCollect { get; set; }
         /// <summary> List of tones that will stop recognizing. </summary>
```

Comment:
What does this mean?

Reply:
Which part?
MaxTonesToCollect = the maximum number of tones that we are recognizing.
List of tones that will stop recognizing. = The StopTones is a list that will stop the recognize operation once recognized.